src/renderer/views/AppSettings/subcomponents/AppUpdate.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { selectVersionDetails } from '../../../../shared/redux/slices/settingsSlice';
+
+export default function AppUpdate() {
+  const { t } = useTranslation();
+  const versionDetails = useSelector(selectVersionDetails);
+  return (
+    <>
+      <Typography>
+        {t('AppSettings.updates.version')} {versionDetails.version}
+      </Typography>
+
+      {versionDetails.isUpToDate ? (
+        ''
+      ) : (
+        <>
+          <Typography>
+            {t('AppSettings.updates.newVersionAvailable')}
+          </Typography>
+          <Button>{t('AppSettings.updates.update')}</Button>
+        </>
+      )}
```

Comment:
@LostInStatic this is a bit awkward. If you want to render something or nothing then you can use the `&&` operator like so:

```ts
{!versionDetails.isUpToDate && (
  <>
    <Typography>
      {t('AppSettings.updates.newVersionAvailable')}
    </Typography>
    <Button>{t('AppSettings.updates.update')}</Button>
  </>
)}
```

Reply:
Must've left that one over from my first draft of the UI. Fixed.